axelor-project/src/main/resources/domains/Project.xml

```diff
@@ -94,7 +90,7 @@
       help="Activates a Planification tab on the project-task-form allowing to record planned times on tasks, in order to compare with actual spent time."/>
     <boolean name="isShowFrequency" title="Task recurrence"
       help="Activate a panel tab, within a project-task-form related to the project, allowing to replicate a task following a date frequency defined by the user."/>
-    <boolean name="isShowTimeSpent" title="Time Spent?"
+    <boolean name="manageTimeSpent" title="Time Spent?"
```

Comment:
Can you add default="true" on this field since it will be used as the old imputable please